packages/attribute-registry-process/src/services/attributeRegistryService.ts

```diff
@@ -42,112 +34,6 @@ export function attributeRegistryServiceBuilder(
   const repository = eventRepository(dbInstance, attributeEventToBinaryData);
 
   return {
-    async createDeclaredAttribute(
-      apiDeclaredAttributeSeed: ApiDeclaredAttributeSeed,
-      authData: AuthData
-    ): Promise<AttributeId> {
-      logger.info(
-        `Creating declared attribute with name ${apiDeclaredAttributeSeed.name}}`
-      );
-      if (authData.externalId.origin !== "IPA") {
-        throw originNotCompliant("IPA");
-      }
-
-      return unsafeBrandId<AttributeId>(
-        await repository.createEvent(
-          createDeclaredAttributeLogic({
-            attribute: await readModelService.getAttributeByName(
-              apiDeclaredAttributeSeed.name
-            ),
-            apiDeclaredAttributeSeed,
-          })
-        )
-      );
-    },
-
-    async createVerifiedAttribute(
-      apiVerifiedAttributeSeed: ApiVerifiedAttributeSeed,
-      authData: AuthData
-    ): Promise<AttributeId> {
-      logger.info(
-        `Creating verified attribute with name ${apiVerifiedAttributeSeed.name}`
-      );
-      if (authData.externalId.origin !== "IPA") {
-        throw originNotCompliant("IPA");
-      }
-
-      return unsafeBrandId<AttributeId>(
-        await repository.createEvent(
-          createVerifiedAttributeLogic({
-            attribute: await readModelService.getAttributeByName(
-              apiVerifiedAttributeSeed.name
-            ),
-            apiVerifiedAttributeSeed,
-          })
-        )
-      );
-    },
-    async getCertifierId(tenantId: TenantId): Promise<string> {
-      const tenant = await readModelService.getTenantById(tenantId);
-      if (!tenant) {
-        throw tenantNotFound(tenantId);
-      }
-
-      const certifier = tenant.data.features
-        .filter(({ type }) => type === "Certifier")
-        .find(({ certifierId }) => certifierId.trim().length > 0);
-
-      if (certifier) {
-        return certifier.certifierId;
-      }
-      throw OrganizationIsNotACertifier(tenantId);
-    },
```

Comment:
Additional refactor.I think the function `getCertifierId` should not be inside `attributeRegistryServiceBuilder` object, because this object only holds functions that are directly exposed to the router. So I moved the `getCertifierId` function outside that object.